components/link/link.js

```diff
@@ -180,7 +181,7 @@ class Link extends LocalizeCoreElement(FocusMixin(LitElement)) {
 				?download="${this.download}"
 				href="${ifDefined(this.href)}"
 				target="${ifDefined(target)}"
-			><span style="white-space: nowrap;"><span style="white-space: normal;"><slot></slot></span>${newWindowIndicator}${newWindowMessage}</span></a>`;
+			><span style="white-space: nowrap;"><span style="white-space: ${getComputedStyle(this).whiteSpace || 'normal'};"><slot></slot></span>${newWindowIndicator}${newWindowMessage}</span></a>`;
```

Comment:
I think we can make a slight optimization and apply `white-space: nowrap;` to the `<a>`, removing one `<span>`?